lesson4.md

```diff
@@ -228,9 +349,16 @@ if (day === 'Saturday') {
 }
 ```
 
+* First, JavaScript checks whether day is equal to `'Saturday'`. If that is the case, it writes `'Do all the shopping'` to console and _skips_ all the `else if` and `else` branches.
+* The first condition in the `if..else-if...else` that is `true` will be executed, all other code blocks will be skipped.
```

Comment:
```suggestion
* The block in the `if..else-if...else` compound whose condition evaluates to true **first** will be executed, all other code blocks that come after it will be skipped.
```